Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -1927,7 +1927,7 @@
             'Geometry/ForwardSimData/data/zdcsens.xml',
         ],
         4 : [
-            'Geometry/ForwardSimData/data/zdcProdCuts.xml',
+            'Geometry/ForwardSimData/data/zdcProdCuts/2021/v2/zdcProdCuts.xml',
```

Comment:
In this case, the old geometry file, Geometry/ForwardSimData/data/zdcProdCuts.xml, will not be used anymore. Should it be deleted? I am not sure the policy on deleting an unused file in this case. Could you please comment @cvuosalo @kpedro88 ?

Reply:
@srimanob No, the previous version is still in use. It is in the Conditions DB as part of the geometry DB records. Our policy is to retain old versions of geometry files that reflect previous geometries that are stored in the DB. Since we keep old Global Tags and old geometries in the DB for a long time (e.g., Run 1), we also have to keep the corresponding XML file versions.

Reply:
@srimanob, the policy is following: never edit xml from old production releases, only add never version of file. In this way we have frozen DB geometries and we can redo some special production with modified geometry from XML using alternative geometry configurations. 

For Phase-2 there is no DB geometry but similar rule is used. We may modify xml only if it is a fresh file not included into production release.